README.developer.md

```diff
@@ -387,3 +388,7 @@ subdirectory:
 ```bash
 SKIP_BUILDDEP=yes ARTIFACTS_DIR=${PWD}/output ./build-scripts/build-rpm.sh
 ```
+
+### Debian
+
+Creating the BlueChi Debian package is described in the [debian/README.md](./debian/README.md).
```

Comment:
Same with the directory, I'd rather use `dpkg` subdirectory

Reply:
As far as I know, the package tooling like `dpkg-buildpackage` expects a `debian` directory to contain these packaging files... so renaming would break it.

Reply:
OK, so then let's have it as is